weblate/addons/migrations/0002_remove_addon_project_scope_addon_project_and_more.py

```diff
@@ -0,0 +1,49 @@
+# Copyright © Michal Čihař <michal@weblate.org>
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def populate_project_from_component(apps, schema_editor):
+    """Migrate Addons with project_scope from using component to using project."""
+    Addon = apps.get_model("addons", "Addon")
+    for addon in Addon.objects.all():
+        if addon.component and addon.project_scope:
```

Comment:
Not needed, `project_scope` filtering should be done in the datababase (see above suggestion) and component can not be NULL at this point:

```suggestion
```

Reply:
Thanks for pointing this out 👍